### ISSUE TITLE
Can not download https

### ISSUE THREAD

**Issue by naviocean** [leeroybrun/node-mt-files-downloader#18]
Hello,
I've tested, it works fine with http but https doesnt support even when I set port is 443 in options.
ex: https://wordpress.org/latest.zip
Please update and fix this issue
Thanks

**Comment by timomitt2610** [comment]
Hello, @leeroybrun 

I am having the same issue. When trying to download from https, I get following error: "The .mtd file is corrupt. Start a new download.". Is there any workaround for https?

**Comment by karvapallo** [comment]
Yeah I'm having the same issue. Did anyone find a solution for this?

**Comment by timomitt2610** [comment]
You could check https://github.com/zeltice/zeltice-mt-downloader

**Comment by tadeork** [comment]
I think you can do this
`dl.setOptions({     
        port: 443
    });`
